.circleci/config.yml

```diff
@@ -456,6 +462,53 @@ jobs:
       - run: bazel build src/... --build_tag_filters=-docs-package,-e2e --define=compile=aot
       - run: bazel test src/... --build_tag_filters=-docs-package,-e2e --test_tag_filters=-e2e --define=compile=aot
 
+  # ----------------------------------------------------------------------------
+  # Job that runs all Bazel tests against material-components-web#master.
+  # ----------------------------------------------------------------------------
+  mdc_snapshot_test_cronjob:
+    <<: *job_defaults
+    resource_class: xlarge
+    environment:
+      GCP_DECRYPT_TOKEN: *gcp_decrypt_token
+      MDC_REPO_URL: "https://github.com/material-components/material-components-web.git"
+      MDC_REPO_BRANCH: "master"
```

Comment:
I found out on Friday that MDC's development HEAD is actually a branch called `develop` instead of `master`, so I believe we should test that

cc @abhiomkar

Reply:
Yeah [this document](https://github.com/material-components/material-components-web/blob/9349deb54bc2ebc332aadf51d3673c5d9192c388/docs/open_source/releases-and-branches.md#branches) describes that `develop` includes breaking-changes. Though I'm not sure which one we want to pick. It looks like both do not really intersect. e.g. a non-breaking change is not merged into both `master` and `develop` (based on my observation on the repo)